src/code/z_collision_check.c

```diff
@@ -7,7 +7,57 @@ void func_8005B280(GraphicsContext* gfx, Vec3f* vA, Vec3f* vB, Vec3f* vC) {
 }
 
 // draw poly
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_collision_check/func_8005B2AC.s")
+void func_8005B2AC(GraphicsContext* gfx, Vec3f* vA, Vec3f* vB, Vec3f* vC, u8 r, u8 g, u8 b) {
+    Vtx* vtxTbl;
+    Vtx* vtx;
+    f32 nx, ny, nz, originDist;
```

Comment:
for the time being we avoid multi line declares, incase we need to end up shuffling stack order later

Reply:
also name gfxCtx proper